f_check

```diff
@@ -330,6 +330,9 @@ if ($link ne "") {
 	    $flags =~ s/\@/\,/g;
 	    $linker_L .= "-Wl,". $flags . " " ;
 	}
+	if ($flags =~ /-lgomp/ && $CC =~ /clang/) {
```

Comment:
Should be `$ENV{"CC"}` instead of `$CC`, there's no `$CC` defined in the script

Reply:
Right, sorry.

Reply:
Looks like this is still there

Reply:
Meh. I had thrown it out but apparently I put it back a week later with #3045 (inadvertently). Bad local fork hygiene I suppose.

Reply:
and fixed again by #3118, thanks for catching this